MemoTesi/main.tex

```diff
@@ -0,0 +1,341 @@
+
+\begin{table}[H]
+    \centering
+    \begin{tabular}{|c|c|}
+    \hline
+        \textbf{Pratiche agili} & \textbf{Studenti}  \\ \hline
+        Sprint planning & 73  \\ \hline
+        Pair programming & 73  \\ \hline
+        Retrospettiva & 48  \\ \hline
+    \end{tabular}
+    \caption{Tabella semplice (anche questo scritto nell'indice delle tabelle)}
+    \label{tab:simple}
+\end{table}
+
+\subsection{tabelle avanzate}
+Con multirow (e multicolumn che però serve meno) puoi fare righe (colonne) più grandi del normale.\\
+\begin{table}[H]
+    \centering
+    \begin{tabular}{|c|c|c|c|c|}
+    \hline
+        \textbf{Team} & \textbf{LoC verificate} & \textbf{LoC sviluppatori} & \textbf{Ore sviluppatori} & \textbf{LoC/h}  \\ \hline
+        \multirow{2}*{1} & 1148& m: 888& m: 40& 22\cr & Diff: -1852& $\sigma$: 371& $\sigma$: 27& \\ \hline
+        \multirow{2}*{2}&1858& m: 1404& m: 65& 22\cr &  Diff: -448& $\sigma$: 1222& $\sigma$: 78& \\ \hline
+        \multirow{2}*{3}&1640& m: 1400& m: 96& 15\cr &  Diff: -2810& $\sigma$: 1417& $\sigma$: 41& \\ \hline
+    \end{tabular}
+    \caption{CAPTION}
+    \label{tab:avanz}
+\end{table}
+
+\subsubsection{Tabelle girate}
+Se usi landscape la tabella viene girata (nel caso dovessi inserirne una molto grande)
+\begin{landscape}
+\begin{table}[H]
+    \centering
+    \begin{tabular}{|c|c|}
+    \hline
+        \textbf{Pratiche agili} & \textbf{Studenti}  \\ \hline
+        Sprint planning & 73  \\ \hline
+        Pair programming & 73  \\ \hline
+        Retrospettiva & 48  \\ \hline
+    \end{tabular}
+    \caption{Tabella girata}
+    \label{tab:girata}
+\end{table}
+
+\end{landscape}
+
+
+\chapter{Altri comandi}
+bla bla
+\section{Math mode}
+Per inserire simboli matematici (e lettere greche) serve la math mode:
+
+Usando il simbolo del dollaro hai la math mode inline: $5 \times \alpha = 3\lambda$
+
+Altrimenti hai quella con le barre e le quadre \[ \frac{\sum_6^i 3i\theta}{12k^2\times 7}\]
+
+Infine hai quelle con begin equation (che vengono numerate):
+\begin{equation}
+    \frac{1}{2}\times A_{bcd}\times E^{fgh}
+\end{equation}
+
+Anche le equazioni possono avere label.
+\section{url e footnote}
+per mettere un link usa url: \url{wikipedia.it}
+
+per fare note a piè di pagina usa footnote\footnote{Tipo questa}
+\section{verbatim}
+Se ti serve scrivere codice o qualcosa per cui ti serve una formattazione specifica usa verbatim:
+\begin{verbatim}
+    Qui puoi scrivere
+
+    come      vuoi
+    e viene tutto
+
+scritto
+                    monospaziato
+\end{verbatim}
+\section{riferimenti}
+Come detto prima le label servono per riferirsi ad altre parti del testo citate precendentemente.\\
+Ti consiglio di metterle sempre almeno a figure. immagini e capitoli.
+
+Per riferirti a qualcosa basta fare ref seguito dal nome della label, ad esempio "vedi capitolo \ref{chap:intro}".\\In questo modo dal pdf cliccando sulla reference, ti porta direttamente al punto giusto.
```

Comment:
Per le virgolette su LaTeX usare `` in apertura e '' in chiusura, o vengono tipograficamente sbagliate.
```suggestion
Per riferirti a qualcosa basta fare ref seguito dal nome della label, ad esempio ``vedi capitolo \ref{chap:intro}''.\\In questo modo dal pdf cliccando sulla reference, ti porta direttamente al punto giusto.
```